test-data/unit/check-super.test

```diff
@@ -204,6 +204,17 @@ class C(B, Generic[T, S]):
 
 class D(C): pass
 
+[case testSuperInClassMethod]
+class A:
+    def f(self, i: int) -> None: pass
+
+class B(A):
+    @classmethod
+    def g(cls, i: int) -> None:
+        super().f(B(), i)
```

Comment:
Can you also test invalid calls to make sure the arguments are checked properly. Another idea would be to have a narrower argument type in the base class method, and to make sure the correct signature is being used.

Reply:
The narrower arguments would only be relevant when you have a method with the same name in the base class.

Reply:
Done!